launcher/ui/pages/global/APIPage.ui

```diff
@@ -130,6 +130,49 @@
          </layout>
         </widget>
        </item>
+       <item>
+        <widget class="QGroupBox" name="groupBox_minecraft">
+         <property name="title">
+          <string>Minecraft Resource Server</string>
+         </property>
+         <layout class="QVBoxLayout" name="verticalLayout_9">
+          <item>
+           <widget class="QLabel" name="label_11">
+            <property name="text">
+             <string>When certain urls in Minecraft change, you can set new urls here.</string>
+            </property>
+            <property name="textFormat">
+             <enum>Qt::RichText</enum>
+            </property>
+            <property name="wordWrap">
+             <bool>true</bool>
+            </property>
+           </widget>
+          </item>
+          <item>
+           <widget class="QLineEdit" name="resourceURL">
+            <property name="placeholderText">
+             <string/>
+            </property>
+           </widget>
+          </item>
+          <item>
+           <widget class="QLineEdit" name="librariesURL">
+            <property name="placeholderText">
+             <string/>
+            </property>
+           </widget>
+          </item>
+          <item>
+           <widget class="QPushButton" name="applyPropertiesBtn">
+            <property name="text">
+             <string>Apply Meta Server Properties</string>
+            </property>
+           </widget>
```

Comment:
I'm not sure about this button:
- we do not have any apply and save for settings so the name make it stand out(maybe reset)
- also this can change more than just the used props so maybe the location is not quite right.
- how does the user know what configs are changed?